### PR TITLE
completes task 7

### PR DESCRIPTION
**Pull request by CamVale**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -78,4 +78,68 @@ describe("GET /api/articles/:article_id", () => {
...
+    };
+    return request(app)
+      .post(`/api/articles/${pathID}/comments`)
+      .send(newComment)
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe('Not Found')
+      });
+  });
+});
```
Comment (Keb84) [inline __tests__/app.test.js:145]: Another status 404 - username doesn't exist - would be good here 👀

**Review by Keb84 — CHANGES_REQUESTED** [review]
Nice work 🤸 please see my comments, add the tests and resubmit 🚀

**Comment by CamVale** [comment]
Thanks :) I've added the test

**Review by johnhutcheon — APPROVED** [review]
Thanks for making the change Cam, all good to merge 🙂

**Pull request merged**